File: test/py/case59_try_raise.py
def maybe_fail_59(flag: bool) -> int:
    try:
        if flag:
            raise Exception("fail-59")
        return 10
    except Exception as ex:
        return 20
    finally:
        pass


if __name__ == "__main__":
    print(maybe_fail_59(True))

File: test/cs/case59_try_raise.cs
using System;

public static class Program
{
    public static int maybe_fail_59(bool flag)
    {
        try
        {
            if (flag)
            {
                throw new Exception(Exception("fail-59"));
            }
            return 10;
        }
        catch (Exception ex)
        {
            return 20;
        }
        finally
        {
        }
    }

    public static void Main(string[] args)
    {
        Console.WriteLine(maybe_fail_59(true));
    }
}
